src/content/docs/tutorial-customer-experience/improve-customer-experience.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: "Improve your customer experience" 
+metaDescription: "New Relic observability maturity series: this guide gives you a quality foundation for optimizing your web application's performance to improve your customer experience."
+redirects:
+---
+
+After you've [established your current state](/docs/tutorial-customer-experience/establish-current-state/), it's time to take the information you've gathered and use it to start improving your customer experience. Since needs can vary based on each organization, you may have different areas you need to improve. The following steps are a suggested order to use for improving your customers' experience, and you may find that you can skip some steps based on your KPIs. As this is a continuing process, we recommend you bookmark this page so that you can refer to it in the future as your KPIs change. 
```

Comment:
or "established the current state of your digital experience's performance"

Reply:
I'd rewrite this intro so it's less generic and more directed at the reader.

"Only you can know what areas need to be improved. Focus your attention here on what's useful and skip what isn't." or something

Reply:
Excellent suggestion, thank you!